README.md

```diff
@@ -1,105 +1,60 @@
-# Webpack Starter Kit
+# FitLit
 
-## Clone This Repo
+### Overview
 
-That's right, _clone_ not fork. You will use this repo multiple times, but you can only fork a repository once. So here is what you need to do to clone the repo and still be able to push changes to your repo:
+FitLit is meant to help you track your historical health in 3 main categories; Activity, Hydration, and Sleep. You can
+view, track, and see trends of your sleep over all of time or more recently along with the some of the data of your friends.
 
-1. Clone down this repo. Since you don't want to name your project "webpack-starter-kit", you can use an optional argument when you run `git clone` (you replace the `[...]` with the terminal command arguments): `git clone [remote-address] [what you want to name the repo]`
-1. Remove the default remote: `git remote rm origin` (notice that `git remote -v` not gives you back nothing)
-1. Create a new repo on GitHub with the name of `[what you want to name the repo]` to be consistent with naming
-1. Copy the address that you would use to clone down this repo - something like `git@github.com:...`
-1. Add this remote to your cloned down repo: `git remote add origin [address you copied in the previous step]` - do not include the brackets
+### How To Use
 
-Now try to commit something and push it up to your new repo. If everything is setup correctly, you should see the changes on GitHub.
+- Once you are on the splash page of the site you can enter your user ID and then be signed in and taken to the homepage where you will see a panel style layout. The first section under the title will list your account information and the current date.
+**// Add screenshot 1 here //**
 
-## Setup
+- Just below that you will see the form section where you can submit your data for the day by choosing one of the three tabs and submitting the form that appears.
+**// Add screenshot two here //**
 
-After one person has gone through the steps of cloning down this repo and editing the remote, everyone should clone down the repo. 
+- The Steps section will show you:
+    1. If you have hit your step goal for today and a chart comparing your step goal vs. the average step goal amongst all users. You can hover over each section of the chart to display specific numbers for each.
+    2. A ranking of you and your friends based on who has the most steps in the last week.
+    3. Your total positive and negative step trends since you joined. A step trend is defined as any set of three consecutive days where your step count goes up each day or down each day when compared to the previous day.
```

Comment:
I think we should have a screenshot of the Steps section, since we're going to have one for Sleep and Hydration.